src/czml3/properties.py

```diff
@@ -1,7 +1,47 @@
 from .base import BaseCZMLObject
 from .common import Deletable, HasAlignment, Interpolatable
 from .enums import ClockRanges, ClockSteps, LabelStyles
-from .types import Cartesian3Value, Uri
+from .types import Cartesian3Value, RgbafValue, RgbaValue, Uri
+
+
+class Color(BaseCZMLObject, Interpolatable, Deletable):
+    """A color. The color can optionally vary over time."""
+
+    KNOWN_PROPERTIES = ["delete", "rgba", "rgbaf", "reference"]
+
+    def __init__(self, *, delete=None, rgba=None, rgbaf=None):
+
+        if isinstance(rgba, list):
+            rgba = RgbaValue(values=rgba)
+        if isinstance(rgbaf, list):
+            rgbaf = RgbafValue(values=rgbaf)
+
+        self._delete = delete
+        self._rgba = rgba
+        self._rgbaf = rgbaf
+
+    @property
+    def rgba(self):
+        """A color specified as an array of color components [Red, Green, Blue, Alpha]
```

Comment:
Can we have the first line of the docstring in a separate line? (As done with the other properties)